trunk/SUAVE/Components/Energy/Processes/Thrust.py

```diff
@@ -154,57 +156,169 @@ def compute(self,conditions):
         bypass_ratio                = self.inputs.bypass_ratio  
         flow_through_core           = self.inputs.flow_through_core #scaled constant to turn on core thrust computation
         flow_through_fan            = self.inputs.flow_through_fan #scaled constant to turn on fan thrust computation
-        
-        # unpacking from self
+
+        #unpacking from self
         Tref                 = self.reference_temperature
         Pref                 = self.reference_pressure
         mdhc                 = self.compressor_nondimensional_massflow
 
-        
-    
+
+
         ##--------Cantwell method---------------------------------
-        
 
-        # computing the non dimensional thrust
-        core_thrust_nondimensional  = flow_through_core*(gamma*M0*M0*(core_nozzle.velocity/u0-1.) + core_area_ratio*(core_nozzle.static_pressure/p0-1.))
-        fan_thrust_nondimensional   = flow_through_fan*(gamma*M0*M0*(fan_nozzle.velocity/u0-1.) + fan_area_ratio*(fan_nozzle.static_pressure/p0-1.))
-        
+
+        #computing the non dimensional thrust
+        core_thrust_nondimensional  = flow_through_core*(gamma*M0*M0*(core_nozzle.velocity/u0-1) + core_area_ratio*(core_nozzle.static_pressure/p0-1))
+        fan_thrust_nondimensional   = flow_through_fan*(gamma*M0*M0*(fan_nozzle.velocity/u0-1) + fan_area_ratio*(fan_nozzle.static_pressure/p0-1))
+
         Thrust_nd                   = core_thrust_nondimensional + fan_thrust_nondimensional
-      
-     
+
+
         Fsp              = 1./(gamma*M0)*Thrust_nd
 
-        # computing the specific impulse
-        Isp              = Fsp*a0*(1.+bypass_ratio)/(f*g)
-        
-        # computing the TSFC
-        TSFC             = 3600.*f*g/(Fsp*a0*(1.+bypass_ratio))  
-        
-        # computing the core mass flow
+        #Computing the specific impulse
+        Isp              = Fsp*a0*(1+bypass_ratio)/(f*g)
+
+        #Computing the TSFC
+        TSFC             = 3600.*f*g/(Fsp*a0*(1+bypass_ratio))  
+        #computing the core mass flow
         mdot_core        = mdhc*np.sqrt(Tref/total_temperature_reference)*(total_pressure_reference/Pref)
 
-        # computing the dimensional thrust
+        #computing the dimensional thrust
         FD2              = Fsp*a0*(1+bypass_ratio)*mdot_core*no_eng*throttle
 
-     
-        
-        # fuel flow rate
+
+
+        #fuel flow rate
         a = np.array([0.])        
-        fuel_flow_rate   = np.fmax(0.1019715*FD2*TSFC/3600.,a) #use units package for the constants
-        
-        # computing the power 
+        fuel_flow_rate   = np.fmax(0.1019715*FD2*TSFC/3600,a) #use units package for the constants
```

Comment:
These numbers should be replaced so that the source is clear. In general though you should leave the decimal. This is good practice to avoid issues with integers.

Reply:
So add 1/g and unit conversion for the 3600?

Reply:
I left it how it is for now, as I did not alter/use this portion of code.  I used evaluate stream thrust.

Reply:
Make this an issue

Reply:
Done